src/components/hoc/withMeeting.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {concatMap} from 'rxjs/operators';
+import {AdapterContext} from '../hooks/contexts';
+
+/**
+ * @param MeetingComponent 
```

Comment:
let's add a bit more documentation:
- function description
- type for the MeetingComponent parameter (React.Component)
- a short usage example

Reply:
...and return type. See an example here: https://terrestris.github.io/react-geo/docs/@terrestris/react-geo/5.6.2/HigherOrderComponent_VisibleComponent_VisibleComponent.jsx.html